src/pages/components/map-related/MapWithLocationsAndPath.tsx

```diff
@@ -24,30 +21,15 @@ export interface Path {
   vehicleRef: number
 }
 
-interface MapProps {
+export interface MapProps {
   positions: Point[]
   plannedRouteStops: BusStop[]
 }
```

Comment:
exporting this to a different file (`map-types.ts`?) can solve the circular dependency

Reply:
ok